### PR TITLE
[Merged by Bors] - improve error messages

### PR DESCRIPTION
**Pull request by gregwebs**
I was messing around with the initialization of Fluvio and improving these error messages helped me understand what was going on. Also just noticed a typo in an error message.

### CODE REVIEW

**Review by tjtelan — CHANGES_REQUESTED** [review]
This looks good, but CI has some failures. It looks like CI code formatting checks failed, but I expect that crate version checks will also fail.

Can you run `cargo fmt` for the formatting check, and bump the patch version of the publishable crates?

**Review by sehz — CHANGES_REQUESTED** [review]
Thanks for PR.  LGTM.  One minor nit

File: crates/fluvio-spu/src/config/cli.rs [hunk 1]
```diff
@@ -227,7 +227,7 @@ fn find_spu_id_from_env() -> Result<SpuId, IoError> {
         } else {
             Err(IoError::new(
                 ErrorKind::NotFound,
-                "No Spu Id is founded from env",
+                format!("SPU index id not found form SPU_INDEX or {FLV_SPU_ID}")
```
[the remark this answers is not in the document]
Comment (gregwebs) [reply crates/fluvio-spu/src/config/cli.rs]: oh, yes! PR is updated now.

**Comment by gregwebs** [comment]
I ran `cargo fmt`.

> and bump the patch version of the publishable crates

Oh, is there some automation to help with that?

**Comment by tjtelan** [comment]
> I ran `cargo fmt`.
> 
> > and bump the patch version of the publishable crates
> 
> Oh, is there some automation to help with that?

There isn't, but [it looks like the test passed](https://github.com/infinyon/fluvio/runs/7866880300?check_suite_focus=true), so there's nothing you need to do. (Someone recently updated these crates too, so they updated the versions for next release.)

However, it looks like the [clippy](https://github.com/infinyon/fluvio/runs/7866879840?check_suite_focus=true) test failed

**Review by tjtelan — APPROVED** [review]

**Review by sehz — APPROVED** [review]
LGTM.  Thanks for cleanup

**Pull request closed without merging**